test/distributed/org/apache/cassandra/fuzz/topology/TopologyMixupTestBase.java

```diff
@@ -536,14 +640,25 @@ public String ipAddress(int nodeNum)
                     };
                 }
             });
+            commandsTransformers.add((state, commandGen) -> rs2 -> {
```

Comment:
to make the logs easier to read, every command has a pre/post log you can jump to